soroban-sdk/src/ledger.rs

```diff
@@ -75,6 +75,11 @@ impl Ledger {
 
     /// Returns a unix timestamp for when the ledger was closed.
     ///
+    /// The close time is a UNIX timestamp indicating when the ledger closes.
+    /// Its accuracy depends on the system clock of the validator proposing the
+    /// block. Consequently, consensus may confirm a close time that lags a few
```

Comment:
What do you think about simplifying the last sentence? A Soroban developer might not understand the point about consensus.

Reply:
Yeah that's a good point. I think we can change it to:

> Consequently, the close time proposed by a validator may be earlier or later than the actual time, by a few seconds past and up to 60 seconds ahead of other validators.

But I'd like to also be able to say what guarantees we can have about the value relative to other values seen. i.e. Can we say that the value is always increasing?

I've asked about this here:
- https://stellarfoundation.slack.com/archives/C9H5JTW1J/p1720738688324639

Reply:
Updated.